--- a/setup.py
+++ b/setup.py
@@ -224,6 +224,11 @@
         ('build_man_page', None), ('build_icons', None), ('build_translations', None),
     ]
 
+    def run(self):
+        if not os.path.isdir('build'):
+            os.mkdir('build')
+        build.run(self)
+
 
 class raphodo_sdist(sdist):
     def run(self):
@@ -265,8 +270,8 @@
         'tornado',
         'scandir;python_version<"3.5"',
         'typing;python_version<"3.5"',
-        'PyGObject'
-        'PyQt5'
+        'PyGObject',
+        'PyQt5',
     ],
     extras_require={
         'color_ouput': ['colorlog',],
